membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -40,18 +42,18 @@ class AccountController extends LazyLogging {
 
     // TODO - refactor to use the Zuora-only based lookup, like in AttributeController.pickAttributes - https://trello.com/c/RlESb8jG
 
-    val updateForm = Form { single("stripeToken" -> nonEmptyText) }
+    val updateForm = Form { tuple("stripeToken" -> nonEmptyText, "publicKey" -> text) }
```

Comment:
what does `tuple` do? is it just to make the types explicit?

Reply:
It's just saying you're looking to process 2 form-data fields